addons/mrp/models/mrp_production.py

```diff
@@ -575,7 +575,6 @@ def _get_finished_move_value(self, product_id, product_uom_qty, product_uom, ope
             'product_uom': product_uom,
             'operation_id': operation_id,
             'byproduct_id': byproduct_id,
-            'unit_factor': product_uom_qty / self.product_qty,
```

Comment:
your commit message is not satisfying

The next commits will allow changing the initial demand of confirmed moves as well as adding move once the production order is confirmed. In order to not duplicate again the unit factor computation in multiple write and create, make the field a stored computed.